src/http-gateways/trustless-gateway.md

```diff
@@ -235,6 +235,25 @@ In case both are present in the request, the value from the [`Accept`](#accept-r
 
 Below MUST be implemented **in addition** to "HTTP Response" of :cite[path-gateway].
 
+## Response Status Codes
+
+### `200` OK
+
+The request succeeded.
+
+If the HTTP method was `GET`, then data is transmitted in the message body.
+
+### `404` Not Found
+
+Error to indicate that requested resource is not provided by the gateway.
+
+### `3XX` Redirects
```

Comment:
Same, unsure how this plays with #491 and path-gateway status codes. Should we edit there instead?

Begs the question, does it make sense that a recursive trustless gateway redirects and the non-recursive doesn't?